src/library/base/man/NA.Rd

```diff
@@ -25,7 +25,8 @@
   other atomic vector types which support missing values: all of these
   are \link{reserved} words in the \R language.
 
-  The generic function \code{is.na} indicates which elements are missing.
+  The generic function \code{is.na} indicates which elements are missing
+  (\code{NA}) or  (NA) or \sQuote{Not a Number} (\code{link{NaN}}). 
```

Comment:
why "(\code{NA}) or (NA) or ..."? i.e., what's the difference between these two cases (other than the font they'll be set in?)

Reply:
typo, fixed